frontend/src/router/index.ts

```diff
@@ -11,6 +12,10 @@ const router = createRouter({
       path: '/',
       redirect: '/home'
     },
+    {
```

Comment:
A view and route to create what?

Reply:
This route exists for me to create roles to be easier to test the code and also for the code acceptance done by claudio.